docs/language/functions/README.md

```diff
@@ -0,0 +1,51 @@
+# Functions
+
+Functions appear in expression context and
+take Zed values are arguments and produce a value as a result:
```

Comment:
I think this would read better with a period instead of a colon since it doesn't really introduce the list that follows.
```suggestion
take Zed values as arguments and produce a value as a result.
```